src/app/app.component.ts

```diff
@@ -7,4 +8,13 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'Susper';
+  constructor(private router: Router) { }
+  ngOnInit() {
+    this.router.events.subscribe((evt) => {
+      if (!(evt instanceof NavigationEnd)) {
```

Comment:
there could be a case when it is not at the end of the navigation,(somewhere in the middle). this should work in those cases too.